samza-core/src/test/java/org/apache/samza/context/TestTaskContextImpl.java

```diff
@@ -0,0 +1,76 @@
+package org.apache.samza.context;
+
+import java.util.Set;
+import java.util.function.Function;
+import org.apache.samza.Partition;
+import org.apache.samza.checkpoint.OffsetManager;
+import org.apache.samza.container.TaskName;
+import org.apache.samza.metrics.MetricsRegistry;
+import org.apache.samza.scheduling.Scheduler;
+import org.apache.samza.storage.kv.KeyValueStore;
+import org.apache.samza.system.SystemStreamPartition;
+import org.apache.samza.table.TableManager;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+
+public class TestTaskContextImpl {
+  private static final TaskName TASK_NAME = new TaskName("myTaskName");
+
+  @Mock
+  private Set<SystemStreamPartition> _systemStreamPartitions;
```

Comment:
Coding convention: in Samza, we don't use '_' prefix for private members.

Reply:
Ah yes, I think I had caught that, but hadn't updated the PR yet. Thanks for pointing it out anyways.